jobs/dd-agent/templates/config/bpm.yml.erb

```
@@ -0,0 +1,122 @@
+processes:
+  - name: datadog-agent
+    workdir: /var/vcap/jobs/dd-agent
+    executable: "/var/vcap/jobs/dd-agent/bin/agent_bpm"
+    additional_volumes:
+      - path: /var/vcap/data/garden/garden.sock
+      - path: /var/vcap/sys/run/dd-agent
+        writable: true
+      - path: /var/vcap/jobs/dd-agent/packages/dd-agent/run
+        writable: true
+        allow_executions: true
+      - path: /var/vcap/packages/dd-agent/run
+        writable: true
+        allow_executions: true
+    limits:
+<% if p("bpm.agent.memory") != "" %>
+      memory: <%= p("bpm.agent.memory") %>
+<% end %>
+<% if p("bpm.agent.open_files") %>
+      open_files: <%= p("bpm.agent.open_files") %>
+<% end %>
+<% if p("bpm.agent.processes") %>
+      processes: <%= p("bpm.agent.processes") %>
+<% end %>
+    capabilities: [NET_ADMIN, SYS_PTRACE]
+    unsafe:
+      host_pid_namespace: true
+      unrestricted_volumes:
+        - path: /sys/kernel
+        - path: /sys/fs/cgroup
+<% if p('dd.process_agent_enabled', false) == true || p('dd.process_agent_enabled', false).to_s =~ (/(true|t|yes|y|1)$/i) %>
+  - name: dd-agent-process-agent
```

```suggestion
  - name: process-agent
```
nit- what do you think of naming like this?

---

For backwards compatibility, I decided to use the same process names as in [monit config](https://github.com/DataDog/datadog-agent-boshrelease/blob/feeb9f41cc6ec1faa755f2dc992bc8a891a68f61/jobs/dd-agent/monit#L40).

---

makes sense, thanks!